subscription/upgrade.md

```diff
@@ -60,6 +60,6 @@ To update the billing information for your organization or personal account:
 
     > **Note**
     >
-    > The details displayed in the **Contact Information** section is used to generate your invoice. All billing-related information is sent to the email address listed in this section.
+    > The details displayed in the **Contact Information** section is used to generate your invoice. All billing-related information is sent to the email address listed in this section. Invoices are sent to one email address. Please contact your admistrator to create a group within your organization that includes all of the email addresses.
 
 4. Click **Update** to save your changes.
```

Comment:
Add a blank line at the end of the markdown file.

Reply:
Updated